hack/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# This script assumes its running from a pod and does the following:
+# * Processes the template to setup a local cincy instance
+# * Prepares a graph using test data
+# * Ensures generated graph is valid
+
+# Debug information
+oc whoami
+oc project
+
+# Apply oc template
+oc process -f ../dist/openshift/cincinnati.yaml \
+  -p IMAGE=${CINCINNATI_E2E_IMAGE:-pipeline} \
+  -p IMAGE_TAG=${CINCINNATI_E2E_IMAGE_TAG:-deploy} \
+  -p GB_CINCINNATI_REPO="redhat/openshift-cincinnati-test-public-manual"
+
+# Wait for dc to rollout
+oc wait --for=condition=available deploymentconfig/cincinnati
+
+# Check that policy engine returns channel data respond
+curl --verbose --header 'Accept:application/json' http://cincinnati-policy-engine:8080/v1/graph\?channel=a
```

Comment:
How will `http://cincinnati-policy-engine` work with parallel tests?  Won't we need this to use a dynamic, injected job-name or some such to avoid collisions?  Or is this magically running in the same Pod where the Cincinnati-under-test is running?

Reply:
Every test (in Prow) runs in separate namespaces, so it would use the service from this namespace.

Perhaps base URL should be a config option so that e2e could be run on dev machines?

Reply:
Fixed URL (reads `CINCINNATI_PE_BASE_URL`) and PE port